etc/cumulus-config/cumulus-config.yml

```diff
@@ -659,6 +690,13 @@ cumulus_flavors:
   - "{{ cumulus_flavor_lsst_medium }}"
   - "{{ cumulus_flavor_bm_dirac25y }}"
 
+cumulus_flavor_octavia_small:
+  name: "octavia.small"
+  ram: 1024
+  disk: 20
```

Comment:
Sadly this means we will run out of local disk, tiny was the smallest unit I could manage without wasting RAM with the size of our disk :(

Reply:
Well, the DIB image spat out for the amphorae has virtual size 5GB, so we should be able to take the disk on the flavor down to that to save on local disk.